unit/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: all cprover.dir test
 
-SRC = catch_entry_point.cpp \
+SRC = unit_tests.cpp \
```

Comment:
My understanding is that the test is catch_example, which is only specified once, in SRC. That's where extra tests will be added. The other executables here will be migrated to the Catch framework over time and be removed from this makefile.